openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -40,12 +63,13 @@ def __init__(self, *args, **kwargs):
             data[key] = value
 
         data["fps"] = lib.collect_animation_data(fps=True)["fps"]
```

Comment:
Unrelated to this PR. Wasn't this attribute removed a very long time ago? :D Haha, does `fps` really make sense on a maya review instance right since there's a predefined scene fps that we can rely on?

Reply:
I'm almost certain this was used on a cartoon production that was animated on 2s at some point. But I can't for life of me remember the exact use case

Reply:
@mkolar I'd like to mark this conversation resolved so that the PR is ready to merge - however I'm still dieing to know more about that 2s scenario you described and why this was needed.

I'll still mark this resolved since it's not directly related to this PR, but if you have any time to elaborate it'd be lovely.